fiftyone/core/session.py

```diff
@@ -363,6 +402,42 @@ def url(self):
 
         return "http://localhost:%d/" % self.server_port
 
+    @property
+    def color_pool(self):
+        """A list of browser supported color strings from which the App should
+        draw from for coloring fields.
+        """
+        return self.state.color_pool
+
+    @color_pool.setter
+    @_update_state()
+    def color_pool(self, color_pool):
+        self.state.color_pool = color_pool
+
+    @property
+    def show_attrs(self):
+        return self.state.show_attrs
+
+    @show_attrs.setter
+    @_update_state()
```

Comment:
I agree that not auto-showing is the right approach. Nor would I want the App to automatically refresh in-place, in case I'm modifying multiple settings.

But, how hard would it be to make `session.refresh()` reflect these settings? I tried calling `session.refresh()` after changes `session.show_confidence` and didn't see any changes. But I'm unsure if that's because `session.refresh()` is not a full refresh like cmd+R, or if this is a current limitation of refreshing when expanded sample view is open.

Basically I'm wondering if now is a good time to make refreshing the App respect whether the expanded sample view is open :)